lib/Dialect/SV/SVOps.cpp

```diff
@@ -1944,6 +1946,318 @@ void BindOp::build(OpBuilder &builder, OperationState &odsState, StringAttr mod,
+
+  Region emptyRegion;
+  hw::module_like_impl::printModuleSignatureNew(
+      p, emptyRegion, getModuleType(), getAllPortAttrs(), getAllPortLocs());
+
+  SmallVector<StringRef> omittedAttrs = {
+      SymbolTable::getSymbolAttrName(),   SymbolTable::getVisibilityAttrName(),
+      getModuleTypeAttrName().getValue(), getPerPortAttrsAttrName().getValue(),
+      getPortLocsAttrName().getValue(),   getParametersAttrName().getValue()};
+
+  mlir::function_interface_impl::printFunctionAttributes(p, *this,
+                                                         omittedAttrs);
+}
+
+ParseResult SVVerbatimModuleOp::parse(OpAsmParser &parser,
+                                      OperationState &result) {
+  using namespace mlir::function_interface_impl;
+  auto builder = parser.getBuilder();
+
+  // parse optional visibility
+  (void)mlir::impl::parseOptionalVisibilityKeyword(parser, result.attributes);
+
+  StringAttr nameAttr;
+  if (parser.parseSymbolName(nameAttr, SymbolTable::getSymbolAttrName(),
+                             result.attributes))
+    return failure();
+
+  // parse optional parameters
+  ArrayAttr parameters;
+  if (parseOptionalParameterList(parser, parameters))
+    return failure();
+
+  // parse module-like op signature
+  SmallVector<hw::module_like_impl::PortParse> ports;
+  TypeAttr modType;
+  if (failed(
+          hw::module_like_impl::parseModuleSignature(parser, ports, modType)))
+    return failure();
+
+  if (failed(parser.parseOptionalAttrDict(result.attributes)))
+    return failure();
+
+  result.addAttribute("module_type", modType);
+  result.addAttribute("parameters", parameters);
+
+  SmallVector<Attribute> portAttrs, portLocs;
+  for (auto &port : ports) {
+    if (port.attrs && !cast<DictionaryAttr>(port.attrs).empty())
+      portAttrs.push_back(port.attrs);
+    else
+      portAttrs.push_back(builder.getDictionaryAttr({}));
+
+    if (port.sourceLoc && !isa<UnknownLoc>(*port.sourceLoc))
+      portLocs.push_back(*port.sourceLoc);
+    else
+      portLocs.push_back(builder.getUnknownLoc());
+  }
+
+  if (!portAttrs.empty())
+    result.addAttribute("per_port_attrs", builder.getArrayAttr(portAttrs));
+  if (!portLocs.empty())
+    result.addAttribute("port_locs", builder.getArrayAttr(portLocs));
+
+  // parse verbatim content
+  if (!result.attributes.get("content")) {
+    return parser.emitError(parser.getCurrentLocation(),
+                            "sv.verbatim.module requires 'content' attribute");
+  }
+
+  // parse output file
+  auto outputFileAttr = result.attributes.get("output_file");
+  if (!outputFileAttr) {
+    return parser.emitError(
+        parser.getCurrentLocation(),
+        "sv.verbatim.module requires 'output_file' attribute");
+  }
+  if (!isa<hw::OutputFileAttr>(outputFileAttr)) {
+    return parser.emitError(
+        parser.getCurrentLocation(),
+        "sv.verbatim.module 'output_file' attribute must be an OutputFileAttr");
+  }
```

Comment:
nit: Please use a single brace. 
```suggestion
  if (!result.attributes.get("content")) 
    return parser.emitError(parser.getCurrentLocation(),
                            "sv.verbatim.module requires 'content' attribute");
  

  // parse output file
  auto outputFileAttr = result.attributes.get("output_file");
  if (!outputFileAttr) 
    return parser.emitError(
        parser.getCurrentLocation(),
        "sv.verbatim.module requires 'output_file' attribute");
  
  if (!isa<hw::OutputFileAttr>(outputFileAttr)) 
    return parser.emitError(
        parser.getCurrentLocation(),
        "sv.verbatim.module 'output_file' attribute must be an OutputFileAttr");
  
```